services/base-jdbc/src/main/resources/org/eclipse/hono/service/base/jdbc/store/device/base.sql.yaml

```diff
@@ -171,3 +171,11 @@ countDevicesOfTenant: |
    SELECT COUNT(*) AS deviceCount FROM %1$s
    WHERE
       tenant_id=:tenant_id
+
+getDevicesOfTenant: |
+   SELECT * 
+   FROM %s
+   WHERE
+      tenant_id=:tenant_id
+   LIMIT :page_size
+   OFFSET :page_offset
```

Comment:
would you mind adding another line break at the end of the file?

Reply:
done